content/millennium/r4/other/binary.md

```diff
@@ -9,7 +9,7 @@ title: Binary | R4 API
 
 ## Overview
 
-The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to Continuity of Care Documents (CCD), clinical documents and diagnostic reports. The diagnostic reports are currently limited to Anatomic Pathology and Radiology.
+The Binary resource can contain any clinical content such as text, images, and PDFs. This resource is currently limited to clinical documents, diagnostic reports, and communication message content. Diagnostic reports currently supports Anatomic Pathology, Cardiology, Microbiology, and Radiology.
```

Comment:
Wasn't sure on the wording here after the rebase

Reply:
I think we should keep Continuity of Care Documents (CCD)

Reply:
I would leave CCD in the list, just easier to see buzz words in the list.

Also: 'The supported diagnostic reports are...'

Reply:
Added back CCD and I checked with Mayur; cardio and micro should be included on this list.

https://github.com/cerner/fhir.cerner.com/pull/607/commits/99c1be98f1d5fe1d1fa6c1cfa9b11d4c798094b9#diff-b9330978e62a15a61d6612ae8dc779e3b4680761c2dfc7e9b3778909d2d95cd8R12

Reply:
+1